composer.json

```diff
@@ -23,5 +23,11 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5"
```

Comment:
Note to self: consider switching to `symfony/phpunit-bridge` so tests can be run on PHP 7.1 as well. This can overlap with adding a GitHub action for CI builds.

Reply:
Is there anything else you want me to fix or is this PR good to go? :)

Reply:
If you want to take a crack either of the two "Note to self" todo items I left, feel free. I realize you won't be able to setup GitHub CI for the repo, but I don't think there's any roadblock for integrating `symfony/phpunit-bridge`.

If not, not worries. I'll try to get to both times in the next couple of days and hopefully have this merged and tagged in a new release within the week.

Reply:
I'll look into it!

Reply:
Have you been able to take a look at the CI?

Reply:
I opened https://github.com/jmikola/AutoLogin/pull/24 to test CI and it appears to be working. It looks like we need to bump the minimum dependency of symfony/security to 4.4.1 as the AbstractListener class was only added in https://github.com/symfony/symfony/commit/b20ebe6b908cadca5c37040f305a101ce8cdf981. That was a bit of a surprise as I wouldn't have expected a new file to be introduced in a patch release.

Let me know if those changes look good and if so I'll squash and merge everything ASAP.

Reply:
> That was a bit of a surprise as I wouldn't have expected a new file to be introduced in a patch release.

Yes, usually they are very good at not making big changes in  patch releases.

The CI looked good, I left a few suggestions/questions. 

Should I close this PR?

Reply:
We can leave this PR open for now. I'm just using the other PR for the CI work, so I'll plan to squash/merge this and then rebase to add the final changes for CI.